tests/Test/AbstractIntegrationTestCase.php

```diff
@@ -143,6 +143,8 @@ public function testIntegration(IntegrationCase $case): void
     {
         foreach ($case->getSettings()['deprecations'] as $deprecation) {
             $this->expectDeprecation($deprecation);
+            $this->expectDeprecation($deprecation);
+            $this->expectDeprecation($deprecation);
```

Comment:
Does it make sense to expect exactly the same deprecation message multiple times? I would rather want to ensure that every expected deprecation message happens at least once. What is the gain from checking the exact amount of messages?

Reply:
:+1:

we care to detect that there is deprecation. There is no value in having same deprecation msg once or 10 times.

Reply:
> Does it make sense to expect exactly the same deprecation message multiple times?

Of course not, I just wanted to show that if we would want to check the exact count (as suggested in https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7578#discussion_r1434021534) there would be multiple expectations of the same message.

Reply:
that was totally _not_ my point @kubawerlos . 
my point was _what if yet another deprecation would happen_. let say, the "unexpected" one - we should fail. now we silently move on.